main.go

```diff
@@ -23,6 +28,17 @@ func main() {
 	// Parse command-line arguments
 	flag.Parse()
 
+	// Tracing can be configured independently of the orchestration code.
+	tp, err := ConfigureZipkinTracing()
```

Comment:
Should this be under some flag like `use-zipkin` or some other name? In case user is using any collector other than zipkin?

Reply:
This change was the wrong commit, I will revert it. Thanks for catching it. This PR will not include changes for setting up the exporter, it only includes changes for injecting trace info to activity.